From 8f672215f756236aa36868726fbbbdab5cc6bf2b Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 13 Jun 2016 12:06:19 +0800
Subject: [PATCH 1089/5242] MLK-12900-2: ARM: dts: fix asrc mqs noise issue

commit  6ad1eeb964d097153bd14923835aa58677f201c4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change the spdif's clock frequency for the frequency is too high
for asrc, which will cause the asrc p2p + mqs noise.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
(cherry picked from commit e623ad471a46cbe7d12f1ae8a58d5ea53129db22)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
index 8ff24da..b7c3afe 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-wm8958.dts
@@ -132,7 +132,7 @@
 	assigned-clocks = <&clks IMX6UL_CLK_SPDIF_SEL>,
 			  <&clks IMX6UL_CLK_SPDIF_PODF>;
 	assigned-clock-parents = <&clks IMX6UL_CLK_PLL4_AUDIO_DIV>;
-	assigned-clock-rates = <0>, <98304000>;
+	assigned-clock-rates = <0>, <49152000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

